src/Image/index.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { CSSProperties } from 'react';
+
+interface ImgProps {
+    alt: string;
+    height?: number;
+    src: string;
+    title: string;
+    width?: number;
```

Comment:
What would cause `height` and `width` to be `undefined`?

Reply:
This was a change I had added to work with making `img` responsive, but forgot to take back out.